qabelbox/src/main/java/de/qabel/qabelbox/index/AndroidIndexSyncService.kt

```diff
@@ -58,6 +60,7 @@ class AndroidIndexSyncService() : IntentService(AndroidIndexSyncService::class.j
             }
         } catch (ex: Throwable) {
             error("Error syncing with index. Action: ${intent.action}", ex)
+            crashSubmitter.submit(ex)
```

Comment:
crash reporter install a crash-handler into the android runtime. The crash-submitter submits an exception as a crash. This is for cases where we catch an exception, but it is still unexpected.